cmds/ps/ps.go

```diff
@@ -168,17 +174,61 @@ func (pT *ProcessTable) PrepareString() {
 	pT.fstring = fstring
 }
 
+func mapSubset(check map[byte]int, ref map[byte]int) bool {
+	for cLet, cNum := range check {
+		if rNum, ok := ref[cLet]; ok {
+			if cNum != rNum {
+				return false
+			}
+		} else {
+			return false
+		}
+	}
+	return true
+}
+
+func isPermutation(check string, ref string) bool {
```

Comment:
Maybe you can sort two character arrays and make sure they are the same:
Here is a short program I wrote demonstrating how:
``package main

import (
	"fmt"
	"sort"
	"strings"
	"reflect"
)

func main() {
	s := "stressed"
	y := "desserts"
	runes := strings.Split(s, "")
	runesy := strings.Split(y, "")
	sort.Strings(runes)
	sort.Strings(runesy)
	fmt.Println(reflect.DeepEqual(runes, runesy))
}
``
From documentation : "Array values are deeply equal when their corresponding elements are deeply equal."